packages/messaging/README.md

```diff
@@ -0,0 +1,67 @@
+
+# React Native AEPMessaging Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepmessaging.svg)](https://www.npmjs.com/package/@adobe/react-native-aepmessaging) 
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepmessaging)](https://www.npmjs.com/package/@adobe/react-native-aepmessaging)
+
+## Usage
+
+### [Messaging](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-journey-optimizer)
+
+##### Importing the extension:
+
+```javascript
+import {AEPMessaging} from '@adobe/react-native-aepmessaging';
+```
+##### Registering the extension with AEPCore
+Initialize the Messaging SDK via native code inside Appdelegate and MainApplication for iOS and Android respectively.
```

Comment:
We updated info about how to initialize and registering the extension in the main readme.  Then We can refer the setup from extension's readme.
https://github.com/adobe/aepsdk-react-native/tree/main/packages/userprofile
https://github.com/adobe/aepsdk-react-native/tree/main/packages/edge

Do you want to do the similar for Messaging?

Reply:
`Initialize the Messaging SDK` is a bit confusing here. Can we replace it with `Initialize the SDK and register the Messaging extension ....`?